sympy/core/expr.py

```diff
@@ -47,6 +47,8 @@ class Expr(Basic, EvalfMixin):
 
     is_scalar = True  # self derivative is 1
 
+    args: tuple
```

Comment:
It should be a tuple of Basic. See the same hint in `basic.py`. Is it necessary to add this hint for Expr or can it just inherit the hint from Basic?

Reply:
Indeed unnecessary. I should instead add type hints to sub classes of `Expr`.